### PR TITLE
Mozilla.Firefox version 12.0

### PR DESCRIPTION
**Pull request by Tesco-epic**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/15764)

### CODE REVIEW

**Comment by KaranKad** [comment]
`Architecture` is not `neutral`, its `x86`

**Comment by KaranKad** [comment]
`ShortDescription` is too short.

File: manifests/m/Mozilla/Firefox/12.0/Mozilla.Firefox.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: Mozilla.Firefox
+PackageVersion: 12.0
+Installers:
+- Architecture: neutral
```
Comment (mddtest) [inline manifests/m/Mozilla/Firefox/12.0/Mozilla.Firefox.installer.yaml:6]: - Architecture: x86

**Review by mddtest — APPROVED** [review]

File: manifests/m/Mozilla/Firefox/12.0/Mozilla.Firefox.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: Mozilla.Firefox
+PackageVersion: 12.0
+PackageLocale: en-US
+Publisher: Mozilla
+PackageName: Firefox 12
+License: Mozilla
+ShortDescription: Firefox 12 WIN 32
```
Comment (mddtest) [inline manifests/m/Mozilla/Firefox/12.0/Mozilla.Firefox.locale.en-US.yaml:9]: ShortDescription: a free web browser backed by Mozilla, a non-profit dedicated to internet health and privacy.

File: manifests/m/Mozilla/Firefox/12.0/Mozilla.Firefox.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+
+PackageIdentifier: Mozilla.Firefox
+PackageVersion: 12.0
+Installers:
+- Architecture: neutral
+  InstallerType: exe
+  InstallerUrl: http://ftp.mozilla.org/pub/firefox/releases/12.0/win32/en-GB/Firefox%20Setup%2012.0.exe
+  InstallerSha256: 4C7CB1270A059CD6FE8AFF9DEABD7B27EEEFCD5AA42D2BF0914C7D7BAA191C89
+ManifestType: installer
+ManifestVersion: 1.0.0
```
Comment (mddtest) [inline manifests/m/Mozilla/Firefox/12.0/Mozilla.Firefox.installer.yaml:11]: InstallerSwitches:
    Silent: /silent
    SilentWithProgress: /silent

**Pull request closed without merging**